README.md

```diff
@@ -1,8 +1,2 @@
-# React + Vite
-
-This template provides a minimal setup to get React working in Vite with HMR and some ESLint rules.
-
-Currently, two official plugins are available:
-
-- [@vitejs/plugin-react](https://github.com/vitejs/vite-plugin-react/blob/main/packages/plugin-react/README.md) uses [Babel](https://babeljs.io/) for Fast Refresh
-- [@vitejs/plugin-react-swc](https://github.com/vitejs/vite-plugin-react-swc) uses [SWC](https://swc.rs/) for Fast Refresh
+The website is available at
+[www.news-explorer.serverpit.com](www.news-explorer.serverpit.com)
```

Comment:
* Please, deploy the app, and show the **correct** link in `Readme`


Now it leads to `404 - page not found` 
![image](https://github.com/user-attachments/assets/3e4b1f81-fd7f-4ae6-8bce-6aa48f1bb9aa)